docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+
+- **Common** APIs (introduced with Office 2013) can be used to access features such as UI, dialogs, and client settings that are common across multiple types of Office applications. This object model uses [callbacks](https://developer.mozilla.org/en-US/docs/Glossary/Callback_function). For details about the Common API object model, which includes APIs for interacting with Outlook, see [Office JavaScript API object model](office-javascript-api-object-model.md).
+
+> [!NOTE]
+> Excel Custom functions run within a unique runtime that prioritizes execution of calculations, and therefore uses a slightly different programming model. For details, see [Custom functions architecture](../excel/custom-functions-architecture.md).
+
+For additional information about the Office JavaScript APIs, see [Understanding the JavaScript API for Office](understanding-the-javascript-api-for-office.md).
+
+#### API requirement sets
+
+Requirement sets are named groups of API members. Requirement sets can be specific to Office hosts, such as the `ExcelApi 1.7` requirement set (a set of APIs that can only be used in Excel), or common to multiple hosts, such as the `DialogApi 1.1` requirement set (a set of APIs that can be used in any Office application that supports the Dialog API).
+
+Requirement set support varies by Office host, version, and platform. For detailed information about the platforms, requirement sets, and Common APIs that each Office application supports, see [Office Add-in host and platform availability](../overview/office-add-in-availability.md).
+
+## Creating an Office Add-in 
+
+You can create an Office Add-in by using the Yeoman generator for Office Add-ins or Visual Studio.
+
+[!include[Yeoman vs Visual Studio comparision](../includes/yeoman-generator-recommendation.md)]
+
+### Yeoman generator for Office Add-ins
+
+The [Yeoman generator for Office Add-ins](https://github.com/officedev/generator-office) can be used to create a Node.js Office Add-in project that can be managed with Visual Studio Code or any other editor. The generator can create Office Add-ins for any of the following:
+
+- Excel
+- OneNote
+- Outlook
+- PowerPoint
+- Project
+- Word
+- Excel custom functions
+
+You can choose to create the project using HTML, CSS and JavaScript, or using Angular or React, and can choose between JavaScript and Typescript as well.
```

Comment:
This sentence is confusing to me because of the "and" and "or" groupings. Consider breaking this into two sentences: one for framework and one of language.